text/dependencies/rfcs/0002-dependency-server-purl.md

```diff
@@ -0,0 +1,48 @@
+# Add pURL Field to the Dependency Server Metadata
+
+## Summary
+
+Add a field for pURLs into the dependency server schema.
+This RFC is an addendum to [RFC #0009](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0009-dep-server.md)
+
+## Motivation
+
+Currently, the dependency server offers a lot of valuable information about the
+files it hosts. However, it currently does not have the capacity to supply
+pURLs. This field should be added to the dependency server because pURLs are an
+industry standard package identifier that allow dependencies to be accurately
+identified by the end user.
```

Comment:
```suggestion
pURLs. This field should be added to the dependency server because pURLs are an
industry standard package identifier that allow dependencies to be accurately
identified by the end user. Package URLs are also a standard supported by the CycloneDX Bill of Materials format, and are [integrated with a number of other tools](https://github.com/package-url/purl-spec#users-adopters-and-links). Overall, package URLs can provide rich information beyond what's available via CPEs alone, making them a strong choice to include on our dependency metadata.
```